test/recipe/admin/pages/specs/data-table.vue

```diff
@@ -101,6 +101,41 @@ const variants = [
         footer2: "30"
       }
     })
+  },
+  {
+    name: "Export",
+    descriptor: giveMe.aDataTable({
+      columns: [
+        giveMe.aDataTableColumn({ title: "Label", prop: "label", minWidth: true, component: giveMe.aConditional({ testId: "label" }), exportable: true }),
+        giveMe.aDataTableColumn({ title: "Data 1", prop: "data1", component: giveMe.aConditional({ testId: "prop-1" }), exportable: true }),
+        giveMe.aDataTableColumn({ title: "Data 2", prop: "data2", component: giveMe.aConditional({ testId: "prop-2" }), exportable: true }),
+        giveMe.aDataTableColumn({ title: "Data 3", prop: "data3", component: giveMe.aConditional({ testId: "prop-3" }), exportable: true }),
+        giveMe.aDataTableColumn({ title: "Data 4", prop: "data4", component: giveMe.aConditional({ testId: "prop-4" }), exportable: true })
```

Comment:
remove extra columns, it should be just enough stubbing for the spec to test the case